src/test/ui/feature-gates/feature-gate-unsafe_block_in_unsafe_fn.rs

```diff
@@ -0,0 +1,11 @@
+#![deny(unused_unsafe)]
+
+unsafe fn unsf() {}
```

Comment:
Please do some unsafe operation in here to make sure this is allowed with the default level of `unsafe_op_in_unsafe_fn`.

Reply:
This was my placeholder function for an unsafe operation. Shouldn’t I create another test function instead?